src/main/java/org/folio/orders/utils/ResourcePathResolver.java

```diff
@@ -79,6 +80,7 @@ private ResourcePathResolver() {
     apis.put(PO_NUMBER, "/orders-storage/po-number");
     apis.put(PURCHASE_ORDER_STORAGE, "/orders-storage/purchase-orders");
     apis.put(PIECES_STORAGE, "/orders-storage/pieces");
+    apis.put(PIECES_STORAGE_BATCH, "/orders-storage/pieces-batch");
```

Comment:
need to add module permission to call this new endpoint

Reply:
Added permission